fingertip/util/systemd.py

```diff
@@ -0,0 +1,9 @@
+"""
```

Comment:
an entire file for what's a one-liner lambda? nope, please just inline into corresponding os plugins. also, if it were larger, it still wouldn't belong to `utils/`, but rather to `plugins/os/common`

Reply:
I can make it `fingertip/util/initsystems.py` and include the oneliners for OpenRC and Upstart there, if you want?

I could put it into `plugins/os/common`, but then I can't use those from the RHEL OS plugins, can I? Those would also have to duplicate the systemd oneliner, which brings us to 6 copies of the same line (RHEL 7 uses systemd, but needs a different line), which I think is enough to refactor into a separate file.

If you still disagree, I can copy/paste it into all those files, but that feels dirty.

Reply:
I believe this should now be resolved, PTAL.